components/workbenches/workbenches.go

```diff
@@ -0,0 +1,50 @@
+package workbenches
+
+import (
+	"github.com/opendatahub-io/opendatahub-operator/components"
+	"github.com/opendatahub-io/opendatahub-operator/pkg/common"
+	"github.com/opendatahub-io/opendatahub-operator/pkg/deploy"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ComponentName      = "workbenches"
+	Path               = "/opt/odh-manifests/odh-notebook-controller/base"
+	notebookImagesPath = "/opt/odh-manifests/notebook-images/overlays/additional"
```

Comment:
```suggestion
	notebookImagesPath = "/opt/odh-manifests/jupyterhub/notebook-images/overlays/additional"
```
from https://github.com/red-hat-data-services/odh-manifests/tree/feature-rearchitecture/jupyterhub/notebook-images/overlays